src/app/bs-navbar/bs-navbar.component.html

```diff
@@ -1,5 +1,5 @@
 <nav class="navbar navbar-expand-md navbar-light bg-light fixed-top">
-  <a class="navbar-brand" href="#">Navbar</a>
+  <a class="navbar-brand" routerLink="/">O</a>
```

Comment:
whats the 0 for?

Reply:
That's the title of the homepage that Mosh gives.

Reply:
ship it